broker/plugins/kubejobs/plugin.py

```diff
@@ -233,6 +235,70 @@ def errors(self):
             return ()
         return self.rds.lrange("job:errors", 0, -1)
 
+    def validate(self, data):
+        data_model = {
+            "cmd": list,
+            "config_id": six.string_types,
+            "control_parameters": dict,
+            "control_plugin": six.string_types,
```

Comment:
control and monitor parameters should not be required as well. we should fix the pieces of code that use them without checking first instead of making them required (but this can be addressed in a follow-up patch).

Reply:
Done